### ISSUE TITLE
Make "go home" button invisible after the app launched

### ISSUE THREAD

**Issue by Rubyroidas** [kitze/JSUI#53]
Right after I launch the app, home button (in the left top corner) is visible. When I dive down into any project and go back it hides.

**Comment by kitze** [comment]
@Rubyroidas hmm, it makes sense to me to hide the home button when on the home page? Is that what you're saying?

**Comment by Rubyroidas** [comment]
@kitze yes, when app is just started and you're in the home screen. hide it please until user selects any project

**Comment by kitze** [comment]
Oooh, this only happens in the packaged version, I didn't see it in the dev version so that's why I was confused. Thanks for reporting, will fix!

**Comment by jakedohm** [comment]
I opened a branch to fix this yesterday, and then assumed Kitze had already fixed it because the problem wasn't showing up in dev. Glad to know I'm not completely crazy!
